src/Generator/ClassGenerator.php

```diff
@@ -30,6 +30,11 @@ class ClassGenerator extends AbstractGenerator
     protected $namespaceName = null;
 
     /**
+     * @var array Array of string names
```

Comment:
I think this should be `UseGenerator[]`, or `UseStatement[]`. Stringly typed will bite us back here